argo/providers/kfp/internal/provider.go

```diff
@@ -36,6 +31,19 @@ type Parameters struct {
 	GrpcKfpApiAddress        string `yaml:"grpcKfpApiAddress,omitempty"`
 }
 
+type ResourceReferences struct {
+	PipelineName         common.NamespacedName        `yaml:"pipelineName"`
+	RunConfigurationName common.NamespacedName        `yaml:"runConfigurationName"`
+	RunName              common.NamespacedName        `yaml:"runName"`
+	Artifacts            []pipelinesv1.OutputArtifact `yaml:"artifacts,omitempty"`
+}
+
+var kfpApiConstants = struct {
+	KfpResourceNotFoundCode int32
+}{
+	KfpResourceNotFoundCode: 5,
+}
```

Comment:
```suggestion
const KfpResourceNotFoundCode = 5
```

Reply:
correct, changed in 847762891ac9e4c53e697be2fbd16d03bfb70260